client/opts.go

```diff
@@ -51,8 +51,6 @@ type (
 	v2SessionReqInfo struct {
 		addr *refs.Address
 		verb v2session.ObjectSessionVerb
-
-		exp, nbf, iat uint64
```

Comment:
Why those are not required anymore? And the same as lifetime in session token.

Reply:
These fields were never set. I decided to decrease client's business logic and leave token composition on the user side.